backend/composer/api/serializers.py

```diff
@@ -154,23 +155,159 @@ class Meta:
         fields = ("id", "name", "ontology_uri")
 
 
+class ViaSerializerDetails(serializers.ModelSerializer):
+    """Via Serializer with Custom Logic for from_entities"""
+
+    anatomical_entities = AnatomicalEntitySerializer(
+        many=True,
+    )
+
+    from_entities = AnatomicalEntitySerializer(
+        many=True,
+    )
+
+    class Meta:
+        model = Via
+        fields = (
+            "id",
+            "order",
+            "connectivity_statement_id",
+            "type",
+            "anatomical_entities",
+            "from_entities"
+        )
+
+    def to_representation(self, instance):
+        """
+        Custom representation for Via.
+        """
+        representation = super().to_representation(instance)
```

Comment:
@afonsobspinto I suggest to use a methodfield here to compute the previous vias
see https://www.django-rest-framework.org/api-guide/fields/#serializermethodfield

Reply:
Cool suggestion. Check:
[c47bbc3](https://github.com/MetaCell/sckan-composer/pull/189/commits/c47bbc3ec674f6140a8965953ae5e16e7732e06c)

Reply:
I reverted this commit, because the serializer was not including the from_entities in the schema